### ISSUE TITLE
[Suggestion] Implement stylevar option to have floor buttons without " PeTi Socket" (Pretty useful when using reflect cubes)

### ISSUE THREAD

**Issue by LautaroL20** [TeamSpen210/BEE2-items#845]
Screenshot comparision:

![socket](https://cloud.githubusercontent.com/assets/12244376/18630659/6de9e1e4-7e44-11e6-8dab-bb513c22863c.png)

**Comment by TheDarkBomber** [comment]
That's not a socket, it's a platform

**Comment by TeamSpen210** [comment]
Alternatively, place an angled platform on the floor, set to start disabled. That will produce a similar platform, so the cube can reach. Actually different styles may or may not have that platform.

**Comment by LautaroL20** [comment]
Ok.

**Comment by LautaroL20** [comment]
@TeamSpen210  - Thats not very intuitive, actually the PeTi sockets are a bit annoying for some people, it would be nice to having this otion anyways.

**Comment by LautaroL20** [comment]
I think that this doesn´t have to do with stylevar, instead of that, this would be more appropiate to adding an option on the button´s menu "Start inverted" to not including PeTi sockets.

**Comment by LautaroL20** [comment]
Okay, while checking the beemod pallete item´s menu, saw that there´s something like "Alternate versions" Option Box, very situable to export the configured item directly from the bee2.4 interface. So, this is great to saving time on making aditional options on the PeTi menu as "start reserved" or things like that....

![alternate](https://cloud.githubusercontent.com/assets/12244376/19306546/95d11a32-904c-11e6-93be-1c0d1608320d.png)

**Comment by TeamSpen210** [comment]
The difference between the variants option and settings on items is that the former only allows setting it globally, throughout the map. This would be better as a specific item choice or a new item, since only some buttons need to be like that.

**Comment by Luke18033** [comment]
@TeamSpen210 Yes, this should be Start Reversed (or even Start Enabled, since buttons don't use that). The buttons look much better with the base, but sometimes you just need to remove it.

**Comment by LautaroL20** [comment]
An interesting thing about the editor models of these floor buttons is that Valve didn´t made its plataforms, maybe they were thought on releasing them without this but they mysteriously changed their mind? Who knows...

**Comment by TeamSpen210** [comment]
It's probably more that they're intended to be simplified forms of the real versions.
